packages/zoe/src/contractFacet/types.js

```diff
@@ -82,6 +82,7 @@
  */
 
 /**
+ * @template {object} [OR=any]
```

Comment:
I think templatized declarations should say what their template parameters represent. I had to look around to find that `OR` is `offerResults`. Would you mind saying so here?

Reply:
Good call. I didn't realize I could have more comments on this line without confusing tsc.